rllib/agents/dqn/dqn.py

```diff
@@ -34,37 +35,30 @@
     # N-step Q learning
     "n_step": 1,
 
-    # === Exploration ===
-    # Max num timesteps for annealing schedules. Exploration is annealed from
-    # 1.0 to exploration_fraction over this number of timesteps scaled by
-    # exploration_fraction
-    "schedule_max_timesteps": 100000,
+    # === Exploration Settings ===
+    "exploration": {
+        # The Exploration class to use.
+        "type": "ray.rllib.utils.explorations.epsilon_greedy.EpsilonGreedy",
```

Comment:
Oof, could we make this a bit shorter? The default namespace could be "ray.rllib.utils.explorations."

Reply:
Yeah, I guess each class where we call `from_config` on, could have its own module as default path and then try the given as absolute path as well as [default path]+given one.

Reply:
Done. The from_config will now additionally search for the class string in the module and package of the class it's called on (in the case of Policy: the Exploration base class), so if we call inside Policy: `from_config(Exploration, some_config), it'll look for "EpsilonGreedy" automatically in the ray.rllib.utils.exploration package.